src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -150,6 +160,32 @@ export class MarkdownTextTransformer {
 		}
 	}
 
+	/**
+	 * Instantiate modal for use as callout when inserting Link or Image into markdown.
+	 * @param calloutStyle Style of callout passed in to determine which callout is rendered
+	 */
+	private async createCallout(type: MarkdownButtonType, triggerElement: HTMLElement): Promise<string> {
+		const posX = triggerElement.getBoundingClientRect().left;
+		const posY = triggerElement.getBoundingClientRect().top;
+
+		let calloutString: unknown = type === MarkdownButtonType.IMAGE_PREVIEW ? 'IMAGE' : 'LINK';
+		let calloutType = (calloutString as ICalloutType);
```

Comment:
Casting it here isn't doing anything - the type is already a string. Just do this : 

`const calloutType: ICalloutType = type === MarkdownButtonType.IMAGE_PREVIEW ? 'IMAGE' : 'LINK';`

In general you should almost never be using unknown/any types.

Reply:
Corrected.